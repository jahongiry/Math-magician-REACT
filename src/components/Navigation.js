import { Link } from 'react-router-dom';
import classes from './Navigation.module.css';

function Navigation() {
  return (
    <header className={classes.header}>
      <Link className={classes.logo} to="/">
        Math Magicians
      </Link>
      <nav>
        <ul>
          <li>
            <Link to="/">Home</Link>
          </li>
          <li>
            <Link to="calculator">Calculator</Link>
          </li>
          <li>
            <Link to="quote">Quote</Link>
          </li>
        </ul>
      </nav>
    </header>
  );
}

export default Navigation;
