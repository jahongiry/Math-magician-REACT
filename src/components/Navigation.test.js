import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navigation from './Navigation';

it('renders if there is Navigation component', () => {
  const three = renderer
    .create(
      <BrowserRouter>
        <Navigation />
      </BrowserRouter>,
    )
    .toJSON();
  expect(three).toMatchSnapshot();
});
