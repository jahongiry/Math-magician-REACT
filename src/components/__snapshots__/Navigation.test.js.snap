// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders if there is Navigation component 1`] = `
<header
  className="header"
>
  <a
    className="logo"
    href="/"
    onClick={[Function]}
  >
    Math Magicians
  </a>
  <nav>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
